docs/pages/get-started/react-redux.mdx

```diff
@@ -278,7 +278,7 @@ can be nested to create the state that you want.
   servers is the winner.
 
 When using our redux integration you don't need to interact directly with these
```

Comment:
should this be more clear? "don't need" can leave room for interpretation that you can use the Live data structures.

Reply:
Yeah, you could add that you _cannot_ interact directly with these datastructures. Good call!

Reply:
Thank you! Adjusted